hedera-node/test-clients/src/main/resource/bootstrap.properties

```diff
@@ -186,7 +186,7 @@ hedera.recordStream.queueCapacity=5000
 hedera.recordStream.recordFileVersion=6
 hedera.recordStream.signatureFileVersion=6
 hedera.recordStream.logEveryTransaction=false
-hedera.recordStream.compressFilesOnCreation=false
+hedera.recordStream.compressFilesOnCreation=true
```

Comment:
Same here, should `balances.compressOnCreation` be `true`?

Reply:
and same here